install.js

```diff
@@ -8,6 +8,7 @@ connection.query(
 	'points INT,' +
 	'num_lines INT,' +
 	'level INT,' +
+	'pf VARCHAR(200),' +
```

Comment:
Why not rename it to something more clear then though 🤔  Like `profile_pic_url`, `profile_pic_path` or just `profile_pic`. Also, can you guarantee that the url is never longer than 200 characters? 🤔